tests/integration/test_charm.py

```diff
@@ -35,3 +35,29 @@ async def test_filebeat_ok(model: Model, application: Application):
     stdout = output.data["results"].get("Stdout")
     stderr = output.data["results"].get("Stderr")
     assert code == "0", f"filebeat test failed with code {code}: {stderr or stdout}"
+
+
+@pytest.mark.abort_on_fail
+async def test_clustering_ok(model: Model, application: Application):
+    """Deploy the charm together with related charms and scale to two units.
+
+    Assert: the clustering config is valid.
+    """
+    model.wait_for_idle(
+        apps=[application.name], status="active", raise_on_blocked=True, timeout=1000
+    )
+    await application.scale(2)
+    await model.wait_for_idle(idle_period=30, apps=[application.name], status="active")
+
+    wazuh_unit = application.units[0]  # type: ignore
+    output = await wazuh_unit.run("/var/ossec/bin/cluster_control -l", timeout=10)
+    code = output.data["results"].get("Code")
+    stdout = output.data["results"].get("Stdout")
+    stderr = output.data["results"].get("Stderr")
+    assert code == "0", f"cluster test for unit 0 failed with code {code}: {stderr or stdout}"
+    wazuh_unit = application.units[1]  # type: ignore
+    output = await wazuh_unit.run("/var/ossec/bin/cluster_control -l", timeout=10)
+    code = output.data["results"].get("Code")
+    stdout = output.data["results"].get("Stdout")
+    stderr = output.data["results"].get("Stderr")
+    assert code == "0", f"cluster test for unit 1 failed with code {code}: {stderr or stdout}"
```

Comment:
I don't think that's enough to check if the cluster is healthy, I'd either parse the output of the command to check if the 2 nodes are there, or even use the `-i` to get even more information

Reply:
Sounds good